### PR TITLE
1/12

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2024-01-12.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+For those of you that our new, the convention of this email is a joke followed by three topics. The running joke is I hate writing the joke. If you go through the commit history of [the Git repository we store these in](https://github.com/dolthub/weekly-updates), you'll see a lot of commits like this one `git commit -am "Everything but the joke"`. This is one of those weeks. I'm not funny.
```
Comment (macneale4) [inline emails/2024-01-12.md:3]:
```suggestion
For those of you that are new, the convention of this email is a joke followed by three topics. The running joke is I hate writing the joke. If you go through the commit history of [the Git repository we store these in](https://github.com/dolthub/weekly-updates), you'll see a lot of commits like this one `git commit -am "Everything but the joke"`. This is one of those weeks. I'm not funny.
```

Comment (tbantle22) [inline emails/2024-01-12.md:3]:
```suggestion
For those of you that are new, the convention of this email is a joke followed by three topics. The running joke is I hate writing the joke. If you go through the commit history of [the Git repository we store these in](https://github.com/dolthub/weekly-updates), you'll see a lot of commits like this one `git commit -am "Everything but the joke"`. This is one of those weeks. I'm not funny.
```

File: emails/2024-01-12.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### Laravel Works with Dolt
+
+We're on a mission to show that Dolt just works with all your favorite tools in all your favorite languages. This week, [the spotlight](https://www.dolthub.com/blog/2024-01-08-dolt-laravel/) is on [PHP](https://www.php.net/) and [Laravel](https://laravel.com/). Laravel is a popular website builder with a built-in ORM called Eloquent. In [this blog](https://www.dolthub.com/blog/2024-01-08-dolt-laravel/), I add branch and merge features to the Laravel bootcamp app, a Twitter...errr....X clone called Chirper. 
+
+[![Dolt + Laravel](../images/dolt-laravel.png)](https://www.dolthub.com/blog/2024-01-08-dolt-laravel/)
+
+[The demo is open source](https://github.com/dolthub/chirper). It's is cool to play with to show how branches and merges can be used to make a [Version Controlled application](https://www.dolthub.com/blog/2024-01-08-version-controlled-application/).
```
Comment (tbantle22) [inline emails/2024-01-12.md:15]:
```suggestion
[The demo is open source](https://github.com/dolthub/chirper). It's cool to play with to show how branches and merges can be used to make a [Version Controlled application](https://www.dolthub.com/blog/2024-01-08-version-controlled-application/). 
```

File: emails/2024-01-12.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+[![Dolt + Laravel](../images/dolt-laravel.png)](https://www.dolthub.com/blog/2024-01-08-dolt-laravel/)
+
+[The demo is open source](https://github.com/dolthub/chirper). It's is cool to play with to show how branches and merges can be used to make a [Version Controlled application](https://www.dolthub.com/blog/2024-01-08-version-controlled-application/). 
+
+### Building a Version Controlled Application
+
+Speaking of which, [Zach](https://www.dolthub.com/team#stephanie) wrote [an article](https://www.dolthub.com/blog/2024-01-08-version-controlled-application/) on what a version controlled application is and how to build one with Dolt. If you want to add an audit log, branches, merges, diffs, and pull requests to your application, Dolt is the way.
```
Comment (tbantle22) [inline emails/2024-01-12.md:19]:
```suggestion
Speaking of which, [Zach](https://www.dolthub.com/team#zach) wrote [an article](https://www.dolthub.com/blog/2024-01-08-version-controlled-application/) on what a version controlled application is and how to build one with Dolt. If you want to add an audit log, branches, merges, diffs, and pull requests to your application, Dolt is the way.
```

**Review by tbantle22 — APPROVED** [review]

File: emails/2024-01-12.md (hunk 3 above)
Comment (macneale4) [inline emails/2024-01-12.md:19]: "version controlled application" sounds like all software. "Application with version control features" is more specific.

Comment (timsehn) [reply emails/2024-01-12.md]: The blog title is version controlled application. I'm sticking with that.

**Pull request merged**